atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -460,10 +460,24 @@ def add_log_type(output):
                 },
             },
         })
-        annotations = self.workflow.build_result.annotations
-        if annotations and 'digests' in annotations:
-            digests = get_digests_map_from_annotations(annotations['digests'])
-            metadata['extra']['docker']['digests'] = digests
+
+        if self.workflow.push_conf.docker_registries:
```

Comment:
I'd really like to see this whole block in its own method for enhanced readability.

Reply:
ok will do that